src/routes/search.js

```diff
@@ -240,6 +249,33 @@ class SearchRoute extends Component {
     this.search({ ...params, page });
   };
 
+  notFoundMessage = (searchType) => {
+    const { params } = this.state;
+    let idWithQuery; let idnoQuery;
+
+    if (searchType === searchTypes.PROVIDERS) {
```

Comment:
As it's a static data, it can be moved out of the component to a function. To keep component it as thin as possible.

Reply:
Thx-- moved lookup of translation keys to a utilities function